dali/base/dautils.cpp

```diff
@@ -46,18 +46,21 @@
 #define SDS_CONNECT_TIMEOUT  (1000*60*60*2)     // better than infinite
 #define MIN_REDIRECTION_LOAD_INTERVAL 1000
 
-
-bool isHostInPlane(IPropertyTree *plane, const char *host, bool ipMatch)
+static IPropertyTree *getPlaneHostGroup(IPropertyTree *plane)
 {
     Owned<IPropertyTree> planeGroup;
     if (plane->hasProp("@hostGroup"))
         planeGroup.setown(getHostGroup(plane->queryProp("@hostGroup"), true));
-    else
-    {
-        if (!plane->hasProp("hosts"))
-            return false;
+    else if (plane->hasProp("hosts"))
         planeGroup.set(plane); // plane itself holds 'hosts'
-    }
+    return planeGroup.getClear();
```

Comment:
trivial: setting into 'planeGroup' doesn't really make code clearer, would be more concise to code as:

```
    if (plane->hasProp("@hostGroup"))
        return getHostGroup(plane->queryProp("@hostGroup"), true);
    else if (plane->hasProp("hosts"))
        return LINK(plane); // plane itself holds 'hosts'
```